src/Microsoft.Extensions.WebEncoders.Core/HtmlEncoderWrapper.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text.Encodings.Web;
+
+namespace Microsoft.Extensions.WebEncoders
+{
+    public class HtmlEncoderWrapper : IHtmlEncoder
+    {
+        private readonly HtmlEncoder _encoder;
+
+        public HtmlEncoderWrapper()
+        {
+            _encoder = HtmlEncoder.Default;
+        }
+
+        public HtmlEncoderWrapper(HtmlEncoder encoder)
+        {
+            _encoder = encoder;
+        }
+
+        public HtmlEncoderWrapper(TextEncoderSettings settings)
+        {
+            _encoder = HtmlEncoder.Create(settings);
+        }
+
+        public static HtmlEncoderWrapper Default { get; } = new HtmlEncoderWrapper(HtmlEncoder.Default);
+
+        public string HtmlEncode(string value)
+        {
+            return _encoder.Encode(value);
+        }
+
+        public void HtmlEncode(string value, int startIndex, int charCount, TextWriter output)
```

Comment:
Having parity between the wrapper and BCL types makes sense to me. How much work would it be to do it?